### ISSUE TITLE
Layout/SpaceBeforeBlockBraces false-positive with empty block

### ISSUE THREAD

**Issue by khiav223577** [rubocop-hq/rubocop#8106]
## Expected behavior

Both `7.times{ p 1 }` and `7.times{}` should be valid when `EnforcedStyle` is set to be `no_space`

## Actual behavior

Rubocop complains about it:
```
Layout/SpaceBeforeBlockBraces: Space missing to the left of {.
```

## Steps to reproduce the problem

Add `test.rb`:
```rb
# frozen_string_literal: true

7.times{ p 1 }
7.times { p 1 }

7.times{}
7.times {}
```

And add `.rubocop.yml`
```yml
Layout/SpaceBeforeBlockBraces:
  EnforcedStyle: no_space
  Enabled: true
```

Run `rubocop test.rb` in command line, it will print:
```
test.rb:4:8: C: Layout/SpaceBeforeBlockBraces: Space detected to the left of {.
7.times { p 1 }
       ^
test.rb:6:8: C: Layout/SpaceBeforeBlockBraces: Space missing to the left of {.
7.times{}
       ^
```
## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler. Here's an example:

```
$ rubocop -v
0.85.0
```

**Comment by tejasbubane** [comment]
@khiav223577 Please use the config option `EnforcedStyleForEmptyBraces: no_space`. The default value for empty braces config is `spaces` which is why the cop complains.

**Comment by tejasbubane** [comment]
Is there a way for us to check if config was loaded as default of added by user? That way we could change the default `EnforcedStyleForEmptyBraces` to `no_space` when `EnforcedStyle` is set to `no_space`.

**Comment by khiav223577** [comment]
> Please use the config option `EnforcedStyleForEmptyBraces: no_space`. The default value for empty braces config is `spaces` which is why the cop complains.

Thanks for the information. I didn't know there is `EnforcedStyleForEmptyBraces` option.
It will be nice if the option is listed in the doc:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceBeforeBlockBraces
